definitions/npm/antd_v3.x.x/flow_v0.25.x-/antd_v3.x.x.js

```diff
@@ -247,6 +247,17 @@ declare module "antd" {
     validateFieldsAndScroll(options?: Object, callback?: ValidateCallback): void
   };
 
+  declare interface RcBaseFormProps {
+    wrappedComponentRef?: any;
```

Comment:
`React.Ref`?

Reply:
Followed the way it was defined in antd Form here: https://github.com/ant-design/ant-design/blob/2d8396bad79a086ae57b3ef462e681e81786f02d/components/form/Form.tsx#L193

Reply:
`wrappedComponentRef` is used instead of `withRef` following [this note](https://github.com/react-component/form#note-use-wrappedcomponentref-instead-of-withref-after-rc-form140).

Reply:
Mentionned in antd [Form documentation](https://ant.design/components/form/#Form.create(options)) as well:
``
class CustomizedForm extends React.Component { ... }

// use wrappedComponentRef
const EnhancedForm =  Form.create()(CustomizedForm);
<EnhancedForm wrappedComponentRef={(form) => this.form = form} />
this.form // => The instance of CustomizedForm
`

Reply:
Yes, but `wrappedComponentRef` is forwarded to the inner component `ref`, so it accepts a `React.Ref`. Anyway, let's stick to the lib definitions.

Reply:
We can replace `any` -> `React.Ref` it's a good idea I think even though in antd is using more generic type.